src/go/wsl-helper/cmd/k3s_kubeconfig.go

```diff
@@ -159,7 +160,6 @@ func getClusterIP() (net.IP, error) {
 
 func init() {
 	k3sKubeconfigCmd.Flags().String("k3sconfig", "/etc/rancher/k3s/k3s.yaml", "Path to k3s kubeconfig")
-	k3sKubeconfigCmd.Flags().BoolVar(&rdNetworking, "rd-networking", false, "Enable the experimental Rancher Desktop Networking")
```

Comment:
This still needs to be there, right? Since (in the RD networking case, for the host, i.e. Win32) we still need it to say `127.0.0.1` (which get forwarded via WSL internal forwarding) rather than the the address of `eth0` inside the RD namespace, which should be unreachable by the host…